### PR TITLE
PP 313: Fix testnet deployment

### PR DESCRIPTION
**Pull request by antomor**
## What

- Update the rif-relay dependencies
- Add the package-lock.json
- Create an independent docker-compose
- Update node version used

## Why

- To make it easier the deployment
- To make the build reproducible

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
-FROM node:12-alpine AS compiler
+FROM node:16-alpine AS compiler
 RUN apk add --no-cache build-base git bash
 WORKDIR /usr/src/app
 COPY . ./
-RUN npm i
-RUN npm start
+RUN npm i --no-audit
+# RUN npm start
```
Comment (rafaiovlabs) [inline Dockerfile]: It's better to erase than comment, if the npm start is not needed please erase it.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+    # rif-relay:
```
Comment (rafaiovlabs) [inline docker-compose.yml]: Erase rif-relay service if its not used.

File: Dockerfile (hunk 1 above)
Comment (antomor) [reply Dockerfile]: done

File: docker-compose.yml (hunk 2 above)
Comment (antomor) [reply docker-compose.yml]: done

**Pull request closed without merging**